python/imdict.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
@Inokinoki  i understand that python3 is not always located under `/usr/bin`. but on the other hand, this change would violate https://www.debian.org/doc/packaging-manuals/python-policy/ch-python.html#s-interpreter_loc , or, make the downstream packager's life harder. if you believe that that's not valid concern, i'd happy to merge this PR to unblock you.

Reply:
Thanks for info ! I prefer not to violate the pyhton policy from debian.

I'll investigate the SCons build system, to check whether it is possible to set an macOS-specified python interpreter.

Reply:
"#!/usr/bin/python3" is better. openSUSE Community does not suggest "#!/usr/bin/env python3".  You will get a error from rpmlint check like this:

```
[ 1045s] ukui-kwin.x86_64: E: env-script-interpreter (Badness: 9) /usr/share/kconf_update/ukui-kwin-5.18-move-animspeed.py /usr/bin/env python3
[ 1045s] This script uses 'env' as an interpreter. For the rpm runtime dependency
[ 1045s] detection to work, the shebang #!/usr/bin/env python  needs to be patched into
[ 1045s] #!/usr/bin/python  otherwise the package dependency generator merely adds a
[ 1045s] dependency on /usr/bin/env rather than the actual interpreter /usr/bin/python.
[ 1045s] Alternatively, if the file should not be executed, then ensure that it is not
[ 1045s] marked as executable or don't install it in a path that is reserved for
[ 1045s] executables.
```

Reply:
> "#!/usr/bin/python3" is better. openSUSE Community does not suggest "#!/usr/bin/env python3". You will get a error from rpmlint check like this:
> 
> ```
> [ 1045s] ukui-kwin.x86_64: E: env-script-interpreter (Badness: 9) /usr/share/kconf_update/ukui-kwin-5.18-move-animspeed.py /usr/bin/env python3
> [ 1045s] This script uses 'env' as an interpreter. For the rpm runtime dependency
> [ 1045s] detection to work, the shebang #!/usr/bin/env python  needs to be patched into
> [ 1045s] #!/usr/bin/python  otherwise the package dependency generator merely adds a
> [ 1045s] dependency on /usr/bin/env rather than the actual interpreter /usr/bin/python.
> [ 1045s] Alternatively, if the file should not be executed, then ensure that it is not
> [ 1045s] marked as executable or don't install it in a path that is reserved for
> [ 1045s] executables.
> ```

Thanks for info